### PR TITLE
Review and update the Ingress page

### PR DESCRIPTION
**Pull request by shavidissa**
Review and update the content.  There were minor updates.

### CODE REVIEW

**Comment by shavidissa** [comment]
[WIP]

File: content/en/docs/concepts/services-networking/ingress.md [hunk 1]
```diff
@@ -101,19 +101,19 @@ for directing HTTP traffic.
...
-* A backend is a combination of service and port names as described in the
-  [services doc](/docs/concepts/services-networking/service/). HTTP (and HTTPS) requests to the
-  Ingress matching the host and path of the rule will be sent to the listed backend.
+  load balancer directs traffic to the referenced Service.
+* A backend is a combination of Service and port names as described in the
+  [Service doc](/docs/concepts/services-networking/service/). HTTP (and HTTPS) requests to the
+  Ingress that matches the host and path of the rule are sent to the listed backend.
 
-A default backend is often configured in an Ingress controller that will service any requests that do not
+A default backend is often configured in an Ingress controller to service any request that do not
```
Comment (zacharysarah) [inline content/en/docs/concepts/services-networking/ingress.md]: nit: requests that do not

Comment (shavidissa) [reply content/en/docs/concepts/services-networking/ingress.md]: Thank you! :) Updated.

**Pull request merged**